src/components/Builder/Contact/Contact.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Contact = ({ parent, parentText }) => {
```

Comment:
Could we get a more descriptive name for this component? We have a contact page, so ideally this should be something like `ClubContactSection` or something similar